<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Untitled - Notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/.ico/text_editor.ico</normaloff>:/images/.ico/text_editor.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontal_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="text_edit">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="tabStopDistance">
       <double>40.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
	border-bottom-width: 1px;
	border-bottom-style : solid;
	border-bottom-color: #f0f0f0;
	background-color    :  white;
}</string>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
   </widget>
   <widget class="QMenu" name="menu_format">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="action_font"/>
    <addaction name="action_syntax_highlighting"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_format"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>Font...</string>
   </property>
  </action>
  <action name="action_syntax_highlighting">
   <property name="text">
    <string>Syntax highlighting</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
